iml-agent/src/action_plugins/stratagem/action_filesync.rs

```diff
@@ -92,12 +92,14 @@ async fn archive_fids(
                 .output()
                 .await?;
 
-            let FidItem { fid, data } = fid;
-            result.push(FidError {
-                fid,
-                data,
-                errno: output.status.code().unwrap_or(0) as i16,
-            });
+            if output.status.code().unwrap_or(0) != 0 {
+                let FidItem { fid, data } = fid;
+                result.push(FidError {
+                    fid,
+                    data,
+                    errno: output.status.code().unwrap_or(0) as i16,
```

Comment:
This is now quite unlikely to be `0`.  
```suggestion
                    errno: output.status.code().unwrap_or(1) as i16,
```

Reply:
Done